public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -164,50 +116,36 @@ function FolderGroupAndEvaluationInterval({
 
   return (
     <div>
-      <FolderAndGroup />
+      <FolderAndGroup groupfoldersForGrafana={groupfoldersForGrafana} />
```

Comment:
It seems a bit odd to pass `AsyncRequestState` around like this, it seems like we only access `result` in the child components. Is it possible to refactor this so we only pass the results in `<FolderAndGroup />`?

Reply:
Done!